vogue/adapter/plugin.py

```diff
@@ -15,7 +15,7 @@ def setup(self, db_name : str):
         self.db = self.client[db_name]
         self.db_name = db_name
         self.sample_collection = self.db.sample
-        self.analysis_collection = self.db.analysis
+        self.analysis_collection_sample = self.db.analysis_sample
```

Comment:
I think we decided the oposit ..
```suggestion
        self.sample_analysis_collection = self.db.sample_analysis
```